cfgov/v1/__init__.py

```diff
@@ -58,10 +63,12 @@ def environment(**options):
         'get_protected_url': get_protected_url,
         'get_latest_activities': get_latest_activities,
         'get_snippets': get_snippets,
+        'localtime': template_localtime,
     })
 
     env.filters.update({
         'linebreaksbr': linebreaksbr,
+        'localtime': template_localtime,
```

Comment:
[Django includes it in Django templates as `localtime`, not `template_localtime`](https://docs.djangoproject.com/en/1.10/topics/i18n/timezones/#localtime). Here I'm including it in the Jinja2 environment with the same name Django uses in its templates.